Rakefile

```diff
@@ -24,6 +24,10 @@ if Semian.sysv_semaphores_supported?
     ext.ext_dir = 'ext/semian'
     ext.lib_dir = 'lib/semian'
   end
+  Rake::ExtensionTask.new('semian_cb_data', GEMSPEC) do |ext|
```

Comment:
Agree with Scott here. Can we just have a single `extconf.rb` generating one shared object? Perhaps you can get some inspiration from other larger multi-file Ruby C extensions on how to structure. [1](https://github.com/eventmachine/eventmachine/tree/master/ext) [2](https://github.com/sparklemotion/nokogiri/tree/master/ext) [3](https://github.com/tenderlove/psych/tree/master/ext)